source/system/Numbers.ooc

```diff
@@ -100,7 +100,7 @@ LDouble: cover from long double {
 		result
 	}
 	in: func (range: Range) -> Bool {
-		this >= range min && this < range max
+		this as Int >= range min && this as Int < range max
```

Comment:
```
value := Int minimumValue - 1
value in(Int minimumValue .. Int maximumValue)
```
will return true if we round `this`, so we cannot convert this to Int, but must solver the `(long) range min` problem somehow